assignment_1.rb

```diff
@@ -0,0 +1,24 @@
+def solution(number) 
+  arr = []
+  return number if number < 10
+
+  while number != 0 do 
```

Comment:
Ruby provides `zero?` to check whether given number is zero or not. https://ruby-doc.org/core-2.7.3/Numeric.html#zero-3F-method